packages/table/src/presenters/TableHeaderCellPresenter.js

```diff
@@ -21,6 +25,23 @@ export default function TableHeaderCellPresenter(props) {
     ...otherProps
   } = props;
   const { globalResizeStyles } = otherProps;
+
+  const timeoutIdRef = useRef(null);
+  useEffect(() => {
+    if (onColumnWidthChanged) {
+      if (!column?.isResizing) {
+        return;
+      }
+      if (timeoutIdRef.current) {
+        clearTimeout(timeoutIdRef.current);
+        timeoutIdRef.current = null;
+      }
+      timeoutIdRef.current = setTimeout(() => {
```

Comment:
I've implemented the current functionality without a timeout. globalResizeStyels does not satisfy my needs because this object does not contain all information that I need, and the width value has the string format with px in the end, but I need a number without px.